### ISSUE TITLE
Stuttering audio (OGG/WAV/Voice) with Oculus Runtime

### ISSUE THREAD

**Issue by Knifa** [Neos-Metaverse/NeosPublic#2050]
## Describe the bug
Voice output stutters pretty badly with the Oculus runtime, but ONLY voice. Other audio types seem to be unaffected. Even messages have it.

Seems to happen even if the desktop audio device is selected (i.e., my regular headphone output) so not something headset related, at least for me.

Using `-SteamVR` fixes it it right away in all circumstances.

## Relevant issues
I couldn't find any!

## To Reproduce
1. Start NeosVR with the Oculus Runtime.
2. Try to listen to any voice (i.e., a voice message or anyone).
3. Notice stuttering.
4. Start NeosVR with the SteamVR runtime.
5. Notice no stuttering.

## Expected behavior
No stuttering.

## Screenshots / Video
Sorry this video is cut off but on the right hand side, I'm just playing voice messages.
https://user-images.githubusercontent.com/622619/115160374-bad1a100-a08f-11eb-8144-2987cfcc3bdb.mp4

## Bug information (please complete the following information):
 - How often does it happen: Always with OVR.
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: 2021.4.17.185
 - Neos Platform: Windows 10
 - Link to reproduction World/Item: N/A
 - Regression: YES, I hadn't encountered this before (whichever version was directly before this one). Oculus hasn't updated since, as far as I can tell.
 - 
[oculus.log](https://github.com/Neos-Metaverse/NeosPublic/files/6332178/oculus.log)
[DESKTOP-LU12RET - 2021.4.17.185 - 2021-04-18 21_38_54.log](https://github.com/Neos-Metaverse/NeosPublic/files/6332183/DESKTOP-LU12RET.-.2021.4.17.185.-.2021-04-18.21_38_54.log)

[steamvr.log](https://github.com/Neos-Metaverse/NeosPublic/files/6332179/steamvr.log)
[DESKTOP-LU12RET - 2021.4.17.185 - 2021-04-18 21_41_39.log](https://github.com/Neos-Metaverse/NeosPublic/files/6332182/DESKTOP-LU12RET.-.2021.4.17.185.-.2021-04-18.21_41_39.log)

## Reporters:
@Knifa in Discord.

**Comment by Frooxius** [comment]
Do you have an Audio Output Device selected in Settings in Neos when this happens? Or is it set to default?

- Does it happen to everyone's voice?
-  What if you record your own voice with the microphone tool and play it back? 
- What if someone else records their own voice with the microphone tool and you play it?

**Comment by Knifa** [comment]
It turns out I did have an audio device selected! It affects everyone's voice.

With SteamVR, everything is normal as expected.

With Oculus:
- With audio device set to default, everything is normal as expected.
- With audio device set to something specific, _audio player_ things seem to stutter (i.e., WAVs and OGGs brought in, or recording myself with the microphone tool) but also players voices and voice messages 
- ...HOWEVER the second I spawn something that uses the video player (like an MP3, MP4, etc..) stuttering goes away for everything including those audio players that previously stuttered, voices messages, etc.

I think I was mistaken originally when I said that it only affects voice since when I tested it the first time I was testing it with MP3s and videos brought in, not the little audio players.
